spec/features/admin/admin_deletes_themselves.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+feature "admin deletes themselves" do
+  scenario "and is logged out" do
+    user = create(:user, admin: true)
+    log_in_as user
+
+    visit admin_users_path
+    click_on I18n.t("admin.users.delete.submit")
+
```

Comment:
I would add one or two assertions here for `current_page` and the flash message.

For example: 
```
expect(current_path).to eq new_session_path
```

Reply:
Done in sha: 281932b